packages/flutter_tools/lib/src/commands/drive.dart

```diff
@@ -244,35 +256,41 @@ class DriveCommand extends RunCommandBase {
         );
       }
 
+      final bool isAndroidChrome = browser == Browser.androidChrome;
+      final bool useEmulator = argResults['android-emulator'] as bool;
       // set window size
-      final List<String> dimensions = argResults['browser-dimension'].split(',') as List<String>;
-      assert(dimensions.length == 2);
-      int x, y;
-      try {
-        x = int.parse(dimensions[0]);
-        y = int.parse(dimensions[1]);
-      } on FormatException catch (ex) {
-        throwToolExit('''
+      // for android chrome, skip such action
+      if (!isAndroidChrome) {
+        final List<String> dimensions = argResults['browser-dimension'].split(
+            ',') as List<String>;
+        assert(dimensions.length == 2);
+        int x, y;
+        try {
+          x = int.parse(dimensions[0]);
+          y = int.parse(dimensions[1]);
+        } on FormatException catch (ex) {
+          throwToolExit('''
 Dimension provided to --browser-dimension is invalid:
 $ex
         ''');
-      }
-      final async_io.Window window = await driver.window;
-      try {
-        await window.setLocation(const math.Point<int>(0, 0));
-        await window.setSize(math.Rectangle<int>(0, 0, x, y));
-      } catch (_) {
-       // Error might be thrown in some browsers.
+        }
+        final async_io.Window window = await driver.window;
+        try {
+          await window.setLocation(const math.Point<int>(0, 0));
+          await window.setSize(math.Rectangle<int>(0, 0, x, y));
+        } catch (_) {
```

Comment:
Could you update this to a specific Exception type like `catch Exception`? We're turning on a lint for `catch` without `on` clauses soon

Reply:
I've removed the try catch block since it is no longer needed. It was here for android chrome but we will no longer do this so it should be fine.